### ISSUE TITLE
Create templates for better maintaining

### ISSUE THREAD

**Issue by Dun-sin** [bitburner-official/bitburner-filesync#14]
I saw this project is open to contributions, and it doesn't have Pull request and issue templates, it makes it easier for you to maintain your projects.

Here are links for reference: 
[Pull Request Template](https://learn.microsoft.com/en-us/azure/devops/repos/git/pull-request-templates?view=azure-devops)
[Pull Request and Issue Template](https://bakingclouds.com/multiple-template-choices/)

**Comment by Hoekstraa** [comment]
Good idea, let's wait a little on the ancillary tools to be added; tests, linting, formatting, so that we can add those as requirements to the PR template.

**Comment by Dun-sin** [comment]
Or you can just add them later

**Comment by Hoekstraa** [comment]
Could, but honestly wouldn't know what to add to it otherwise. Feel free to propose something though.

**Comment by Dun-sin** [comment]
> Could, but honestly wouldn't know what to add to it otherwise. Feel free to propose something though.

I see, for issue templates on my projects I add for bugs, features and documentation.
For Pull request I just add section to fill like: changes done, issue number fixed, things checked and screenshots of changes if needed
